.github/workflows/post_release_to_reddit.yml

```diff
@@ -0,0 +1,51 @@
+name: Post Release to Reddit
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: bluwy/release-for-reddit-action@v1.2.0
+      id: reddit
+      with:
+        username: ${{ secrets.REDDIT_USERNAME }}
+        password: ${{ secrets.REDDIT_PASSWORD }}
+        app-id: ${{ secrets.REDDIT_APP_ID }}
+        app-secret: ${{ secrets.REDDIT_APP_SECRET }}
+        subreddit: Python
+        title: Celery {{ github.event.release.tag_name }} Released!
+        comment: ${{ github.event.release.body }}
```

Comment:
@thedrow are we typically including changelog details in the GitHub release? Or planning to? If not, I can remove this comment.

Reply:
A link to the release notes is good enough in my opinion.